src/components/presentational/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -156,6 +183,37 @@ export default function TimeSeriesChart(props: TimeSeriesChartProps) {
         </>
     }
 
+    function createLineChartDefs() {
```

Comment:
Since this is a pretty big function and this chart is already pretty complex, it would be nice to split this into a separate helper file, including the createStopsFromThresholds function, so this file can remain smaller